### PR TITLE
Calculator small fixes and updating unit tests

### PR DESCRIPTION
**Pull request by abrodowicz**
- updated Readme file to look better
- updated calculator code
- updated test units code
- unnecessary files deleted from the repo

### CODE REVIEW

**Comment by asket888** [comment]
Please, add a description to your PR

File: .idea/Calculator.iml [hunk 1]
```diff
@@ -7,4 +7,10 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
+  <component name="PackageRequirementsSettings">
+    <option name="requirementsPath" value="" />
+  </component>
+  <component name="TestRunnerService">
+    <option name="PROJECT_TEST_RUNNER" value="Unittests" />
+  </component>
 </module>
```
Comment (asket888) [inline .idea/Calculator.iml]: Please find the way how to remove temporary files from github (use .gitignore to do so)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+4. Install Pipenv with the command: brew install pipenv
+
+
+To use this terminal calculator follow instructions below:
+1) Open a terminal
+2) Go to location where you want to clone this calculator code
+3) Run 'git clone https://github.com/abrodowicz/Calculator' to clone this repository
+4) Run 'cd calculator' to move to local repository folder
+4) Run command 'python calculator.py' to run the code
+5) Run command 'pipenv shell' to activate the virtual environment
```
Comment (asket888) [inline README.md]: To create a pretty and nice Readme.md file, use this documentation

**Review by asket888 — CHANGES_REQUESTED** [review]

File: .idea/Calculator.iml (hunk 1 above)
Comment (abrodowicz) [reply .idea/Calculator.iml]: Done

**Comment by asket888** [comment]
Everything is super cool. Please merge your pool request to master

**Review by asket888 — APPROVED** [review]

**Pull request merged**